x/stakeibc/keeper/msg_server_submit_tx.go

```diff
@@ -396,3 +365,91 @@ func (k Keeper) GetLightClientTimeSafely(ctx sdk.Context, connectionID string) (
 		return latestTime, true
 	}
 }
+
+// query and update validator exchange rate
+func (k Keeper) QueryValidatorExchangeRate(ctx sdk.Context, msg *types.MsgUpdateValidatorSharesExchRate) (*types.MsgUpdateValidatorSharesExchRateResponse, error) {
```

Comment:
nit: feels like this should live in `x/stakeibc/keeper/msg_server_update_validator_shares_exch_rate.go`

Reply:
See discussion on this PR with Sam. I don't feel strongly between the two, but he made a good point re: the flow is more clear if both fns are in the same place